src/Files.App/Helpers/WMI/ManagementEventWatcher.cs

```diff
@@ -11,7 +11,7 @@ namespace Files.App.Helpers
 	public delegate void EventArrivedEventHandler(object sender, EventArrivedEventArgs e);
 
 	/// <summary>
-	/// A public class used to start/stop the subscription to specific indication source,
+	/// A public sealed class used to start/stop the subscription to specific indication source,
```

Comment:
Why is this changed?

Reply:
Good eye, I changed the classes using Replace All, compiling and seeing what can be `sealed` or not.

Reply:
I'd do that too but is that necessary change or does it make sense to be changed?

Reply:
It's not a necessary change, ManagementEventWatcher is not sealed.